src/Host/Client/Impl/Host/LocalRHostConnector.cs

```diff
@@ -91,8 +92,11 @@ public LocalRHostConnector(string name, string rHome, string rhostDirectory = nu
                     process = Process.Start(psi);
                     process.EnableRaisingEvents = true;
 
+                    //GeneralLog.Write($"Broker start {process.Id}");
```

Comment:
Not currently but they may come useful if more test fail at some point. Prefer them removed?